etl/pipelines/kg_builder_pipeline.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+from pyoxigraph import Store
+
+from etl.models.types import AntiRecommendationKey, RecordKey
+
+
+class KgBuilderPipeline(ABC):
```

Comment:
Probably premature abstraction.

Reply:
That's true, I have removed `KgBuilderPipeline`.